src/modules/claim/claim.types.ts

```diff
@@ -25,6 +25,8 @@ export interface IClaimRequest extends IMessage {
 export interface IClaimIssuance extends IMessage {
   issuedToken?: string;
   onChainProof?: string;
+  claimType?: string;
+  claimTypeVersion?: string;
```

Comment:
Why are these props optional? Doesn't every issued claim have a `claimType` + `claimTypeVersion` at this point?

Reply:
Unfortunately no, just on-chain claim without previous request has these props.
https://github.com/energywebfoundation/iam-client-lib/blob/2ff0fa4c4a2d44cf13f8a919352932791154ebac/src/modules/claims/claims.service.ts#L392